telemetry/definitions/commonDefinitions.json

```diff
@@ -4375,6 +4441,75 @@
                 }
             ]
         },
+        {
+            "name": "codewhisperer_codeScanIssueIgnore",
+            "description": "Called when a code scan issue is ignored",
+            "metadata": [
+                {
+                    "type": "component"
+                },
+                {
+                    "type": "credentialStartUrl",
+                    "required": false
+                },
+                {
+                    "type": "detectorId"
+                },
+                {
+                    "type": "findingId"
+                },
+                {
+                    "type": "ruleId",
+                    "required": false
+                }
+            ]
+        },
+        {
+            "name": "codewhisperer_codeScanIssueIgnoreAll",
+            "description": "Called when all code scan issues of a specific type are ignored",
+            "metadata": [
+                {
+                    "type": "component"
+                },
+                {
+                    "type": "credentialStartUrl",
+                    "required": false
+                },
+                {
+                    "type": "detectorId"
+                },
+                {
+                    "type": "findingId"
+                },
+                {
+                    "type": "ruleId",
+                    "required": false
+                }
+            ]
+        },
+        {
+            "name": "codewhisperer_codeScanIssueRegenerateFix",
+            "description": "Called when a code scan issue fix is regenerated",
```

Comment:
why does this need to be a separate metric vs `codewhisperer_codeScanIssueGenerateFix`, instead of a flag